content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -207,7 +207,7 @@ point to the CA certificate and key.
 ### Managing the kubeadm drop-in file for the kubelet {#kubelet-drop-in}
 
 See the [setup of the kubelet drop-in file for systemd](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/kubelet-integration/#the-kubelet-drop-in-file-for-systemd)
-section for details about the how the kubeadm package ships with configuration for how the kubelet should be run.
+section for details about how the kubeadm package ships with configuration for how the kubelet should be run.
```

Comment:
hi @vorburger. The text needs some tweaks and context. I added back some original text. What do you think about these changes:

The `kubeadm` package ships with a configuration file for running the `kubelet` by `systemd`. Note that the `kubeadm` CLI never touches this drop-in file. This drop-in file is part of the `kubeadm` DEB/RPM package.

* For further information, see ["Managing the kubeadm drop-in file for systemd"](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/kubelet-integration/#the-kubelet-drop-in-file-for-systemd).

Reply:
@kbhawkey you really did mean for that bullet point to be there before "* For further..", right?

Reply:
Yes, but your choice.